test/Npgsql.Tests/ReaderTests.cs

```diff
@@ -1167,46 +1170,58 @@ public async Task GetBytes()
         }
 
         [Test]
-        public async Task GetStream([Values(true, false)] bool isAsync)
+        public async Task GetStreamSecondTimeThrows([Values(true, false)] bool isAsync)
```

Comment:
I was just suggesting a tiny new test that shows that a text or int value can be read as a Stream, nothing more. I wouldn't touch this test - it's a complicated one that tests SequentialAccess behavior as well, etc.

Reply:
Complex tests aren't good tests in terms of unit testing. So this one just tests for an exceptions, while the existing does all other things as before. Moreover, it should be expanded to cover other types too, but a bit later (:

Reply:
I agree that this test is a bit overly-comlicated (because it tests multiple things). I just wouldn't touch it as part of this PR :)

Reply:
Do you mean that these lines of code should be returned to `GetStream`?

Reply:
Sorry, I misunderstood your changes - now it's clearer. The splitting up of the complex test is a good thing.

But taking a closer look at the previous test, it tested more than what is being tested here, no? Your new test makes sure that you can't get a stream if another stream is already open on the column. The previous version checked other stuff too - that going back inside the column fails for SequentialAccess, but is OK otherwise.